NEWS.md

```diff
@@ -84,6 +84,12 @@ Standard library changes
 * `push!(c::Channel, v)` now returns channel `c`. Previously, it returned the pushed value `v` ([#34202]).
 * `RegexMatch` objects can now be probed for whether a named capture group exists within it through `haskey()` ([#36717]).
 * A new standard library `TOML` has been added for parsing and printing [TOML files](https://toml.io) ([#37034]).
+* The `Pkg.BinaryPlatforms` module has been moved into `Base` as `Base.BinaryPlatforms` and heavily reworked.
```

Comment:
Was this ever documented / exposed to users? There is nothing in the Pkg docs about it I think.

Reply:
There are a few other references to it, in `Tectonic`, `BugReporting`, `PlotlyBase`, `NNlib`, and `MethodAnalysis`, but I agree it's not too important.  I mostly just wanted to have something to show for weeks of effort.  :P

Reply:
Show them `@time import Plots` instead or `@time import Gtk`?  :P Will likely impress move.